gef-ui/server/controllers/mandatory-criteria/index.js

```diff
@@ -0,0 +1,45 @@
+import _isEmpty from 'lodash/isEmpty'
+import * as Api from '../../services/api'
+import { parseBool, errorHandler } from '../../utils/helpers'
+
+const getMandatoryCriteria = async function (req, res) {
+  try {
+    const criteria = await Api.getMandatoryCriteria()
```

Comment:
I think it would be worth keeping with the Portal & TFM code style of lower case for api just so we don't have to think about it when we have to switch between different projects.

Reply:
Done